back/boxtribute_server/models/enums.py

```diff
@@ -0,0 +1,46 @@
+"""Enumeration types used in the database."""
+import enum
+
+from peewee import CharField
+
+
+class EnumCharField(CharField):
```

Comment:
FYI @aerinsol This is the custom peewee field type to represent Python enums as varchar on data level.